plugins/mcreator-localization/help/default/recipe/unlocking_items.md

```diff
@@ -0,0 +1 @@
+When an item from this list is collected, the recipe will be unlocked in the recipe book.
```

Comment:
It may be good to specify if any or all items to remove any potential ambiguity of this help tip